SCClassLibrary/Common/Collections/String.sc

```diff
@@ -429,11 +429,16 @@ String[char] : RawArray {
 		^this.primitiveFailed
 	}
 	splitext {
+		var sep = thisProcess.platform.pathSeparator;
```

Comment:
please use `Platform:isPathSeparator` instead. we support using either `/` or `\` as a directory separator on windows, but `Platform:pathSeparator` is only `\` on Windows, so this code would still consider `foo.bar/baz` to have the extension `bar/baz` there. sorry, neither of these things is documented with that information.

Reply:
cool, I'll do that